tests/utilities.py

```diff
@@ -1,14 +1,28 @@
-from yarl import URL
-from typing import Text
-
+import tempfile
+import os
 import rasa.utils.io as io_utils
 
+from yarl import URL
+from typing import List, Text
+
 from rasa.nlu.classifiers.diet_classifier import DIETClassifier
 from rasa.nlu.classifiers.embedding_intent_classifier import EmbeddingIntentClassifier
 from rasa.nlu.selectors.response_selector import ResponseSelector
 from rasa.utils.tensorflow.constants import EPOCHS
 
 
+def write_temp_file(contents, suffix, mode="w+") -> Text:
```

Comment:
`utils.io.create_temporary_file()` does almost the same thing. can we use that instead?